DataFormats/DeepFormats/interface/ChargedCandidateFeatures.h

```diff
@@ -0,0 +1,41 @@
+#ifndef DataFormats_DeepFormats_ChargedCandidateFeatures_h
+#define DataFormats_DeepFormats_ChargedCandidateFeatures_h
+
+#include "DataFormats/Common/interface/CMS_CLASS_VERSION.h"
+
+namespace btagbtvdeep {
+
+class ChargedCandidateFeatures {
+
+  public:
+
+    float ptrel;
+    float puppiw;
+    float VTX_ass;
+
+    float btagPf_trackEtaRel;
+    float btagPf_trackPtRel;
+    float btagPf_trackPPar;
+    float btagPf_trackDeltaR;
+    float btagPf_trackPParRatio;
+    float btagPf_trackSip3dVal;
+    float btagPf_trackSip3dSig;
+    float btagPf_trackSip2dVal;
+    float btagPf_trackSip2dSig;
+
+
+    float btagPf_trackJetDistVal;
+
+    float chi2;
+    float quality;
+
+    float drminsv;
+
+    // for ROOT schema evolution
+    CMS_CLASS_VERSION(10)
```

Comment:
only templated classes need this.
non-templated ones should have the class version defined in classes_def.xml file

Also, all new classes should start from version 3.

Reply:
will remove from non-templates. Regarding the number I read too fast the TWiki guide on versions, will put all new product classes to version 3

Reply:
fixed with https://github.com/cms-sw/cmssw/pull/19893/commits/347c96268c4f6f017384998cee2bdee31372fa64